ceph.spec.in

```diff
@@ -887,14 +883,14 @@ cmake .. \
     -DCMAKE_INSTALL_INCLUDEDIR=%{_includedir} \
     -DWITH_EMBEDDED=OFF \
     -DWITH_MANPAGE=ON \
+    -DWITH_SYSTEMD=ON \
```

Comment:
why moving this line around?

Reply:
because it was caught between two conditionals. In the spec file we prefer to keep conditionalized/non-conditionalized in separate sections. But now that systemd is the default, I have a better idea.

Reply:
Ah, nevermind - I guess developers would not appreciate me changing the `WITH_SYSTEMD` default to ON.

Anyway, the WITH_SYSTEMD move is just a cosmetic cleanup, with no effect on the logic.

Reply:
fair enough.

Reply:
curious, which developers?

Reply:
I don't know for sure, but I thought I might get pushback from developers who run "do_cmake.sh" and "vstart.sh" a lot. They don't need cmake to descend into the `systemd/` directory, but I guess it's harmless if it does? @ktdreyer If you like, I'll open a separate PR.

Reply:
WITH_SYSTEMD PR: https://github.com/ceph/ceph/pull/20404